### ISSUE TITLE
react-icons - Uncaught SyntaxError: Unexpected identifier

### ISSUE THREAD

**Issue by Nipodemos** [pikapkg/cdn#109]
Hello!

I tried to use the pre-bundled streaming imports with a small project in react, and got the error in the title.
Here is the screenshot:

![image](https://user-images.githubusercontent.com/11494727/104793380-0e76da80-5781-11eb-8676-e635f57817e8.png)

**Comment by FredKSchott** [comment]
Can you share what the file looked like? I have a sneaking suspicion it returned "File not found" or something else in english, which the browser then failed to parse as JS

**Comment by Nipodemos** [comment]
Sorry I don't know how to do this. Where can I find that file to post it here?

**Comment by FredKSchott** [comment]
Oh sorry, just going to that URL in the browser that's shown there in the "Source" section. You can open it in a browser to see what the file contents are

**Comment by Nipodemos** [comment]
I did that, but it just says "Malformed URL". I removed the `[1:18]` since it did not looked like part of the url and same error.

![image](https://user-images.githubusercontent.com/11494727/104808464-443db280-57c5-11eb-8e0e-1c3b03b02083.png)

Don't know if it is relevant but here it is the console, showing no errors at all.
![image](https://user-images.githubusercontent.com/11494727/104808519-92eb4c80-57c5-11eb-9dbf-0d7f032adf4e.png)

**Comment by Nipodemos** [comment]
Guys, i just thought this out a little more and maybe found out the problem.

The thing is, when I import a icon, i do like that: 

```js
import { FiLogIn } from 'react-icons/fi';
```

For those who are unfamiliar with this package, they have several differents collections of multiple icons. So in order to import a icon you have to tell of which collection it belongs. In this case, "feather icons", that's where the `fi` comes from.

I don't know if i explained well enough, and sorry for those who already know how this works.


TLDR:

this works:
```js
 import something from 'react-icons';
```

this does not work: 
```js
import { FiLogIn } from 'react-icons/fi'; //because of the 'package/folder' thing i guess
```

**Comment by aloker** [comment]
I get the same issue with core-es when trying to compile a react project.

```
Unhandled Runtime Error
Uncaught SyntaxError: Unexpected identifier

Source
http://localhost:3000/_snowpack/pkg/-/core-js@v3.8.3-o5NwUKO3gXBpFMmX8I2x/es [:1:18]
SyntaxError: Unexpected identifier
```

**Comment by matthewp** [comment]
@aloker You're using streaming imports as well?

**Comment by aloker** [comment]
Yes, streaming imports are enabled. I'll try to analyze what exactly is causing it.

**Comment by drwpow** [comment]
Update: we just pushed an update to the CDN and I’m able to get this working! 🎉 The following code works for me in the latest version of Snowpack:

```js
import { FiLogIn } from 'react-icons/fi';
```

Going to mark this issue as resolved, but please let me know if anyone experiences any issues with this.

**Comment by Nipodemos** [comment]
Thank you guys for the hard work! This error is gone but another appeared in it's place, i'll make another issue
